cli/src/commands/util.rs

```diff
@@ -108,16 +118,26 @@ fn cmd_util_completion(
 fn cmd_util_gc(
     ui: &mut Ui,
     command: &CommandHelper,
-    _args: &UtilGcArgs,
+    args: &UtilGcArgs,
 ) -> Result<(), CommandError> {
     if command.global_args().at_operation != "@" {
         return Err(user_error(
             "Cannot garbage collect from a non-head operation",
         ));
     }
+    match &args.prune {
+        None => {}
+        Some(s) if s.eq("now") => {}
```

Comment:
nit: `Some("now")` can be used with `args.prune.as_deref()`

Reply:
Ah, thanks. Done.